pyproject.toml

```
@@ -7,4 +7,10 @@ profile = "black"
 py_version = 38
 line_length = 100
 combine_as_imports = true
-filter_files = true
\ No newline at end of file
+filter_files = true
+
+[tool.taskipy.tasks]
+black = { cmd = 'task lint black', help = 'Run black' }
+isort = { cmd = 'task lint isort', help = 'Run isort' }
+lint = { cmd = 'pre-commit run --all-files', help = 'Checks all files for linting errors' }
+precommit = { cmd = 'pre-commit install --install-hooks', help = 'Installs the precommit hook' }
```

```suggestion
black = { cmd = "task lint black", help = "Run black" }
isort = { cmd = "task lint isort", help = "Run isort" }
lint = { cmd = "pre-commit run --all-files", help = "Checks all files for linting errors" }
precommit = { cmd = "pre-commit install --install-hooks", help = "Installs the precommit hook" }
```
All the file uses `"` so let's have consistency.